content/docs/start/data-and-model-access.md

```diff
@@ -1,13 +1,16 @@
 ---
-title: 'Get Started: Data Access'
+title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data Access
+# Get Started: Data and Model Access
 
-Okay, now that we've learned how to _track_ data and models with DVC and how to
-version them with Git, next question is how can we _use_ these artifacts outside
-of the project? How do I download a model to deploy it? How do I download a
-specific version of a model? How do I reuse datasets across different projects?
+Okay, now that we've learned how to _track_ data files in DVC and how to version
+them with Git. _Models_ in a machine learning project are also files written and
+read by programs and DVC can track and version them similar to data files.
+
+Next question is how can we _use_ these artifacts outside of the project? How do
```

Comment:
Would be ideal to keep a single paragraph here.

Reply:
Also, "The next questions are ..."

Reply:
Fixed in a3b15ba4e4a463fe47884150fc25238b85853cbe. Thanks. 👍🏼

Reply:
Thanks. But it's still missing "The".

Reply:
Fixed in b46cca31